pkg/storage/rdbms/all_test.go

```diff
@@ -0,0 +1,43 @@
+package rdbms
+
+import (
+	"testing"
+
+	"github.com/gomods/athens/pkg/storage"
+	"github.com/stretchr/testify/suite"
+)
+
+const (
+	baseURL = "testbaseurl"
+	module  = "testmodule"
+	version = "v1.0.0"
+)
+
+var (
+	// TODO: put these values inside of the suite, and generate longer values.
+	// This should help catch edge cases, like https://github.com/gomods/athens/issues/38
+	//
+	// Also, consider doing something similar to what testing/quick does
+	// with the Generator interface (https://godoc.org/testing/quick#Generator).
+	// The rough, simplified idea would be to run a single test case multiple
+	// times over different (increasing) values.
+	mod = []byte("123")
+	zip = []byte("456")
+)
+
+type RDBMSTests struct {
+	suite.Suite
+	storage storage.StorageConnector
+}
+
+func (r *RDBMSTests) SetupTest() {
+
+	store := NewRDBMSStorage("development_postgres")
+	store.Connect()
+	store.conn.TruncateAll()
```

Comment:
can you use `https://github.com/gobuffalo/suite` for this test suite? the basic idea is that your `RDBMSTests` struct would look like this:

```go
type RDBMSTests struct {
    suite.Model // this is the https://github.com/gobuffalo/suite model type
    // ...
}
```

In doing so, you won't have to worry about manually connecting or truncating and things will be a bit cleaner